docs/content/installation/upgrade_manual_openshift.md

```diff
@@ -34,4 +34,8 @@ When using the Red Hat® OpenShift® Container Platform, the CSI (Container Stor
 
 12. Check the **Overview** tab and that the **Controller Image Tab** and **Node Image Tag** are showing the most up-to-date version of the driver and the **Status** is _Running_.
 
+13. (Optional for initial installation during upgrade only) If desired, create the host definer by clicking click **Create Instance** to create the host definer (`IBMBlockCSI_hostdefiner`).
```

Comment:
```suggestion
13. (Optional, if wasn't done before) If desired, create the host definer by clicking click **Create Instance** to create the host definer (`IBMBlockCSI_hostdefiner`).
```
not sure what `initial installation during upgrade only` means - creating the host definer is not for upgrade only, and generally speaking, "upgrade" is not an "initial installation"

Reply:
while upgrade for the _driver_ isn't an initial installation, if someone is upgrading this may be an initial installation for the host definer, especially for 1.10.

text updated